host/fabricshared/ProtocolGateway.Host.Fabric.FabricShared.Security/SecureStrings.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace ProtocolGateway.Host.Fabric.FabricShared.Security
+{
+    #region Using Clauses
+    using System;
+    using System.Runtime.InteropServices;
+    using System.Security;
+    #endregion
+
+    /// <summary>
+    /// Contains common secure string handling methods.
+    /// </summary>
+    public static class SecureStrings
```

Comment:
Changed, but I would suggest there is a value. The value is to protect the developers from themselves incase a script or hand manipulation changes encoding. I have seen it happen in projects